Assets/Scripts/HealthBar.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace RPG.Attributes
```

Comment:
this might affect what scripts/methods you can access, consider nixing this maybe?